### DISCUSSION TITLE
Ender 3 S1 Display Problem

### DISCUSSION THREAD

**Discussion by Stami1966** [mriscoc/Ender3V2S1#1134] [Q&A] [answered]
Screen problem: I don't see any numbers or values on the display of the Ender 3 S1. See video.
Auto Bed Level and Traming Wizard. I also have the current version.
My 2nd Ender 3 S1 works with the same firmware. Why don't I see any numbers?

https://github.com/mriscoc/Ender3V2S1/assets/150626642/4e0f17fd-78b9-45f5-83ca-0c893e73ebb0

**Comment by overnxt** [comment, marked as answer]
Did you read "How to update the display" on the wiki?

There is an known issue with TCJ displays, maybe it is your case. 

https://github.com/mriscoc/Ender3V2S1/wiki/How-to-update-the-display

Already answered here:
https://github.com/mriscoc/Ender3V2S1/discussions/1118

**Comment by Stami1966** [comment]
Thank you very much for the quick help. I thought it was a different display board, when I opened it and compared it to your pictures, it really wasn't the same as my other S1 printers. Thanks to your correct TJC update (Ender3S1-F4-TJC-20230904.bin) it worked wonderfully. Now I also see the values. Big praise for your great firmware! Better than Creality's crap! Thanks again! -:)
![My Display Board](https://github.com/mriscoc/Ender3V2S1/assets/150626642/a2364c2e-9d19-4e9c-a664-7674a8ccd100)
